config.js

```diff
@@ -23,8 +23,8 @@ const config =
 		// NOTE: Set your own valid certificate files.
 		tls        :
 		{
-			cert : process.env.HTTPS_CERT_FULLCHAIN || `${__dirname}/../exokit-backend/cert/fullchain.pem`,
-			key  : process.env.HTTPS_CERT_PRIVKEY || `${__dirname}/../exokit-backend/cert/privkey.pem`
+			cert : process.env.HTTPS_CERT_FULLCHAIN || `${__dirname}/certs/fullchain.pem`,
```

Comment:
Are these certs being uploaded to the repo? Or are they required to be constantly refreshed and private?

Reply:
These certs are automatically uploaded by our `scp` script in main server. So need to refresh and update the 1 main server's certs and it reflects in all *newly created* worlds. If worlds are created say 10 mins before the certs expire... those worlds would be sort of stuck unless we manually went in and fixed them. We could implement certificate renewal logic that SSHs into all of our instances and updates the certs but so far not a feature of the infrastructure.